go/mysql/server_flaky_test.go

```diff
@@ -263,6 +263,8 @@ func getHostPort(t *testing.T, a net.Addr) (string, int) {
 	return host, port
 }
 
+const mysqlVersion = "8.0.30-Vitess"
```

Comment:
@ajm188 I'll fix up the ones I missed, but it can't be used in all cases, since this is in the test file only. That's deliberate so it doesn't get used accidentally.